src/TestSuite/Fixture/FixtureInjector.php

```diff
@@ -21,6 +21,8 @@
 use PHPUnit_Framework_Test;
 use PHPUnit_Framework_TestListener;
 use PHPUnit_Framework_TestSuite;
+use PHPUnit_Framework_Warning;
+
```

Comment:
extra new line here that causes CS build to fail. looks good appart from this :+1: